src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -49,6 +52,17 @@ public function setDebug(bool $debug)
         $this->debug = $debug;
     }
 
+    public function setTwig(Environment $twig)
+    {
+        $this->twig = $twig;
+        $this->setStrictVariables($twig->isStrictVariables());
```

Comment:
as discussed, this should be removed as this is introducing a BC break in terms of behavior